src/javascript/app/japan/knowledge_test/knowledge_test.ui.js

```diff
@@ -4,25 +4,29 @@ const urlForStatic = require('../../../_common/url').urlForStatic;
 
 const KnowledgeTestUI = (() => {
     const createTrueFalseBox = (question, show_answer) => {
-        const qid      = question.id;
-        const true_id  = `${qid}true`;
-        const false_id = `${qid}false`;
+        const qid               = question.id;
+        const true_id           = `${qid}true`;
+        const false_id          = `${qid}false`;
+        const center_text_class = 'center-text';
```

Comment:
since this is just text assignment it can be done outside, e.g. on line 6 so we don't have to keep assigning it every time the function is called